landr.conf.js

```diff
@@ -48,7 +48,7 @@ module.exports = {
         primary: '#00A375'
       }
     },
-    callToActionCommand: 'curl -sfL https://balena.io/install.sh | sh',
+    callToActionCommand: 'curl -sfL https://balena.io/engine/install.sh | sh',
```

Comment:
Is this what we want for this URL? Alison said that the balenaEngine project landing page will be `balena.io/engine`.